src/index.ts

```diff
@@ -39,6 +38,10 @@ export class CdktfProviderProject extends cdk.JsiiProject {
       constructsVersion,
       minNodeVersion,
       jsiiVersion,
+      authorName = "HashiCorp",
```

Comment:
I wouldn't use the existing `authorName` for this (even though it was named like that before). "Ansgar Mertens" could be a valid author name but would not be valid in the NuGet package name. Let's make it more explicit and expose a "nuGetOrg" instead?